### PR TITLE
DA v2: fix default permissions and update migrations scripts

### PR DESCRIPTION
**Pull request by dlpzx**
### Feature or Bugfix
- Bugfix

### Detail
There are some issues with the permissions that appear in the invitation request

Original:

image

Fresh deployment (with mlStudio module disabled):

image

With a pre-existing deployment:

image

old deployment:
```
	Invite other teams
	Add consumption roles
	create networks
	create pipelines
	create notebooks
	Request datasets access
	create datasets
	create redshift ---> removed! already done
	create ML Studio ---> renamed! already done
```

The following are new or wrong permissions in fresh deployments / backwards
```
	List datasets on this environment / LIST_ENVIRONMENT_DATASETS
	Run athena queries / RUN_ATHENA_QUERY
	List datasets shared with this environments (TYPO!) / LIST_ENVIRONMENT_SHARED_WITH_OBJECTS??
	nothing (mlstudio disabled) / LIST_ENVIRONMENT_SGMSTUDIO_USERS
```

This PR includes 3 fixes:
1) `RUN_ATHENA_QUERY`: Good to add, but was not there before so we need to update the description in the migration scrips

2) `LIST_ENVIRONMENT_DATASETS`, `LIST_ENVIRONMENT_SHARED_WITH_OBJECTS`: they are needed by default, so we can add a new list `ENVIRONMENT_INVITED_DEFAULT` and add them there instead of adding them in the list that is used for the toogle menu.

3) `LIST_ENVIRONMENT_SGMSTUDIO_USERS`: this permission is not used, we just need to remove it

In addition some permissions have been renamed. I used the `migrations/versions/4a0618805341_rename_sgm_studio_permissions.py` script as it already handles renames

Testing:
[X] - Local testing of renaming and descriptions
[X] - AWS testing of the permissions that appear on screen
[ ] - AWS testing with an invited group - check that they can list datasets and shares in environment

This is the end result for a deployment with the dashboards modules disabled:
image

### Relates
- V2

### Security
Please answer the questions below briefly where applicable, or write `N/A`. Based on
[OWASP 10](https://owasp.org/Top10/en/).
 `N/A`
- Does this PR introduce or modify any input fields or queries - this includes
fetching data from storage outside the application (e.g. a database, an S3 bucket)?
  - Is the input sanitized?
  - What precautions are you taking before deserializing the data you consume?
  - Is injection prevented by parametrizing queries?
  - Have you ensured no `eval` or similar functions are used?
- Does this PR introduce any functionality or component that requires authorization?
  - How have you ensured it respects the existing AuthN/AuthZ mechanisms?
  - Are you logging failed auth attempts?
- Are you using or adding any cryptographic features?
  - Do you use a standard proven implementations?
  - Are the used keys controlled by the customer? Where are they stored?
- Are you introducing any new policies/roles/users?
  - Have you used the least-privilege principle? How?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: backend/dataall/core/permissions/permissions.py [hunk 1]
```diff
@@ -27,7 +27,6 @@
 MANAGE_GLOSSARIES = 'MANAGE_GLOSSARIES'
```
Comment (dlpzx) [inline backend/dataall/core/permissions/permissions.py:27]: @nikpodsh why didn't we include the glossaries permissions under the Catalog module?

Comment (nikpodsh) [reply backend/dataall/core/permissions/permissions.py:27]: We simply forgot 😨

File: backend/dataall/modules/dataset_sharing/services/share_permissions.py [hunk 2]
```diff
@@ -46,13 +46,13 @@
...
 ENVIRONMENT_ALL.append(LIST_ENVIRONMENT_SHARED_WITH_OBJECTS)
 
 RESOURCES_ALL.extend(SHARE_OBJECT_ALL)
 for perm in SHARE_OBJECT_ALL:
     RESOURCES_ALL_WITH_DESC[perm] = perm
 
-RESOURCES_ALL_WITH_DESC[CREATE_SHARE_OBJECT] = 'Request datasets access for this environment'
-RESOURCES_ALL_WITH_DESC[LIST_ENVIRONMENT_SHARED_WITH_OBJECTS] = "List datasets shared with this environments"
+RESOURCES_ALL_WITH_DESC[CREATE_SHARE_OBJECT] = 'Create dataset Share requests for this environment'
+RESOURCES_ALL_WITH_DESC[LIST_ENVIRONMENT_SHARED_WITH_OBJECTS] = 'LIST_ENVIRONMENT_SHARED_WITH_OBJECTS'
```
Comment (nikpodsh) [inline backend/dataall/modules/dataset_sharing/services/share_permissions.py:58]: The description used to be nice :(

**Review by nikpodsh — APPROVED** [review]

File: backend/dataall/modules/dataset_sharing/services/share_permissions.py (hunk 2 above)
Comment (dlpzx) [reply backend/dataall/modules/dataset_sharing/services/share_permissions.py:58]: Yes, but this way we can clearly differentiate those that are plotted in the UI from those that are used internally. I just wanted to be consistent

File: backend/dataall/core/permissions/permissions.py (hunk 1 above)
Comment (dlpzx) [reply backend/dataall/core/permissions/permissions.py:27]: Should we open a new PR for that?

**Comment by dlpzx** [comment]
I tested that a new invited team to the environment can list Datasets and Datasets share in the environment. I think it is ready to be merged

**Pull request merged**